packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -235,6 +236,15 @@ class AppBar extends StatefulWidget implements PreferredSizeWidget {
   /// Defaults to 4, the appropriate elevation for app bars.
   final double elevation;
 
+  /// The material's shape as well its shadow.
+  ///
+  /// If shape is non null, the [borderRadius] is ignored and the material's
+  /// clip boundary and shadow are defined by the shape.
+  ///
+  /// A shadow is only displayed if the [elevation] is greater than
+  /// zero.
+  final ShapeBorder shapeBorder;
```

Comment:
To be consistent with the button classes, Material, etc, this property should be called "shape"

Reply:
Yes,  this property should be called "shape".